tests-e2e/ocw-ci-test-course/video-tabs.spec.ts

```diff
@@ -33,9 +33,19 @@ test("that the Download Button works for multiple embed videos in a page", async
     await videoElement.downloadButton().click()
   }
 })
+
 test("Verify that the 'Download video' and 'Download transcript' links are keyboard navigable and have the correct download URLs", async ({
   page
 }) => {
+  /**
+   * ALERT MAC USERS
+   * =================
+   * This test will only pass if MacOS System setting "Keyboard navigation" is
+   * enabled. This setting is **disablled** by default.
```

Comment:
This is definitely true in Firefox.

I was surprised to learn this is also true in the current nightly build of Chromiun. Maybe this behavior is coming to Chrome, too?